src/app/regular-giving/regular-giving.component.html

```diff
@@ -138,6 +138,89 @@
                 </button>
               </div>
 
+            </mat-step>
```

Comment:
code below mostly copied from donation start form. I don't think there's any nice way to make it be shared, and we may find the two evolve separately.